js/chatbot/shared/ChatBot.svelte

```diff
@@ -215,37 +261,51 @@
 											{line_breaks}
 											on:load={scroll}
 										/>
-									{:else if message !== null && message.file?.mime_type?.includes("audio")}
-										<Audio
-											data-testid="chatbot-audio"
-											controls
-											preload="metadata"
-											src={message.file?.url}
-											title={message.alt_text}
-											on:play
-											on:pause
-											on:ended
-										/>
-									{:else if message !== null && message.file?.mime_type?.includes("video")}
-										<Video
-											data-testid="chatbot-video"
-											controls
-											src={message.file?.url}
-											title={message.alt_text}
-											preload="auto"
-											on:play
-											on:pause
-											on:ended
-										>
-											<track kind="captions" />
-										</Video>
-									{:else if message !== null && message.file?.mime_type?.includes("image")}
-										<Image
-											data-testid="chatbot-image"
-											src={message.file?.url}
-											alt={message.alt_text}
-										/>
-									{:else if message !== null && message.file?.url !== null}
+									{:else if message !== null && "component" in message}
+										{#if message.component == "gallery"}
+											<BaseGallery
```

Comment:
@aliabid94 having trouble dynamically loading and using the different svelte components instead of using this giant if/else block.